fix(content): Various issues in Cognizance (#7261)
endless-sky/endless-sky: data/remnant/remnant 2 cognizance.txt

		personality entering forbearing timid uninterested surveillance staying
		fleet
			names "korath"
			cargo 1

> 			commodities "void sprite parts"

Reviewer: Might be worth noting that there *is* a "void sprite parts" commodity list (see line 869 in `commodities.txt`), and my intent was that this pulls one option from that list to fill a random amount of cargo space.

Having a random 3 cargo doesn't have the same impact.

And yes, while having some void sprite parts is just a junk cargo for the moment, these are intended to be used for future side missions.

Author: It's not just that it adds "junk cargo." It is totally ignored.
If it were given a price range like that normal trade commodities, it would be possible for it to show up here, but that would also mean it appears in the trade tab on every planet that has one, which is nonsensical.

![image](https://user-images.githubusercontent.com/20605679/191547321-3d6766cf-daf6-453d-8d40-6647de045023.png)

Author: I have given them all `cargo 0`, so now they won't have any commodities (and can't have outfits because there are no outfitters near where they spawn).

		personality forbearing timid uninterested surveillance staying
		fleet
			names "korath"

> 			cargo 3

Reviewer: Likewise here: There *is* a "void sprite parts" commodity listed in `commodities.txt`, and this is supposed to pull from that list. If that's not working, I'd like to fix it rather than replace that cargo with random miscellaneous stuff.

		system "Aescolanus"
		fleet
			names "korath"

> 			cargo 3

Reviewer: Likewise, I'd prefer to ensure the intended behavior is happening, rather than replacing it with a non-intended generic behavior.

				goto next
			label fleetkilled
			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant. He waves you forward as soon as you approach and introduces you to the briefing. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and recently destroyed part of their fleet. The Captain will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather wrap around their necks, which extends up to their hairline. The leather has a scaly texture, and it appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
			label next

> 			`	He turns to you. "These are the captains who will be following your lead. They are flying two Albatrosses, two Ibises with full complements, and six Starlings. One of the Albatrosses is outfitted to act as a shield; let it take the brunt of attacks." He pauses, looking around at the group. "This task force would be overkill for the intended target, but we are not taking any chances: make sure every Korath there is killed. Salvage is secondary: priority is making sure there are no escapees."`

Reviewer: The priority is probably more correct, but this is a spoken statement where the Disney had dropped the "the".

		conversation
			branch fleetkilled
				has "remnant: cognizance: killed korath fleet"

> 			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`

Reviewer 2: ```suggestion
			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant. He waves you forward as soon as you approach and introduces you to the briefing. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather wrap around their necks, which extends up to their hairline. The leather has a scaly texture, and it appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
```

Comma absolutely required for the original fix, but this one is a bit less awkward still, as well as smoothing out the other part I found grating.

Reviewer: I think this is an improvement.

> 			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`

Reviewer 2: ```suggestion
			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant, and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
```

Author: Is this supposed to be an Oxford comma or is there some other reason to add it?

Reviewer 2: "He waves you forward..." is a separate independent clause joined with the and conjunction, which requires the comma.

> 			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`

Reviewer: Looks like a good change.

				goto next
			label fleetkilled

> 			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant and he waves you forward as soon as you approach. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and recently destroyed part of their fleet. The Captain will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather neck wrap that extends right up to their hairline. It has a scaly texture and appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`

Reviewer 2: ```suggestion
			`When you arrive in the spaceport, you find Prefect Chilia in the middle of briefing a group of Remnant. He waves you forward as soon as you approach and introduces you to the briefing. "Everyone, this is Captain <last>, who has located the Korath hunters twice now and recently destroyed part of their fleet. The Captain will take the lead on tracking the rest for this mission." As you cross the room to join him, you take a good look at the group. One detail that stands out to you is that every one of the Remnant in this group is wearing a thick leather wrap around their necks, which extends up to their hairline. The leather has a scaly texture, and it appears to be pulsing faintly. Unfortunately, you do not have time to ask questions, as Chilia has not stopped issuing instructions.`
```

Reviewer: This seems good.

			label next

> 			`	He turns to you. "These are the captains who will be following your lead. They are flying two Albatrosses, two Ibises with full complements, and six Starlings. One of the Albatrosses is outfitted to be a shield to take the brunt of the attacks." He pauses, looking around at the group. "This task force would be overkill for the intended target, but we are not taking any chances: make sure every Korath there is killed. Salvage is secondary, and there should be no escapees."`

Reviewer 2: ```suggestion
			`	He turns to you. "These are the captains who will be following your lead. They are flying two Albatrosses, two Ibises with full complements, and six Starlings. One of the Albatrosses is outfitted to act as a shield; let it take the brunt of attacks." He pauses, looking around at the group. "This task force would be overkill for the intended target, but we are not taking any chances: make sure every Korath there is killed. Salvage is secondary: priority is making sure there are no escapees."`
```

> 			`	He turns to you. "These are the captains who will be following your lead. They are flying two Albatrosses, two Ibises with full complements, and six Starlings. One of the Albatrosses is outfitted to be a shield to take the brunt of the attacks." He pauses, looking around at the group. "This task force would be overkill for the intended target, but we are not taking any chances: make sure every Korath there is killed. Salvage is secondary, and there should be no escapees."`

Author: The game largely doesn't capitalise after a colon unless it is required.
There are a few news items, one hail from one Person, several instances in the Remnant content written by Zitchas, and a few occurrences in the Deep missions where optional capitalisation is used.
The vast majority of colons are not followed by capitals unless they have to be (including, as far as I can tell, all the colons used by the original developer) and, even in the files I've mentioned where capitalisation is used, it is not consistent and there are cases where the colon isn't followed by a capital.
I think we should be consistent and not capitalising unless it is necessary is the predominant style within the game.

> 			`	He steps back and looks at the group. "May the Embers burn bright for you." With that, he gives the group a respectful nod and walks away. The captains all look around, and make various signs you think are probably encouraging combat pilot slang and variations on Prefect Chilia's parting words. As they disperse to their ships, one gestures to you. "Lead on, Captain."`

Reviewer 2: ```suggestion
			`	He steps back and looks at the group. "May the Embers burn bright for you." With that, he gives the group a respectful nod and walks away. The captains all look around and make various signs you think are probably encouraging combat pilot slang and variations on Prefect Chilia's parting words. As they disperse to their ships, one gestures to you. "Lead on, Captain."`
```

				accept
	npc kill
		government "Korath"
		personality staying target
		system "Nenia"
		fleet
			names "korath"

> 			cargo 3

Reviewer: Same as previous instances.

	name "Transport engineer to <planet>"
	description "An engineer working for Plume needs transportation for himself and some equipment to <planet>."
	blocked "Plume's engineer needs a free bunk and <tons> of cargo space for his equipment. Come back when you have enough space."
	source "Aventine"
	destination "Esquiline"
	cargo equipment 8
	passengers 1
	to offer
		has "Remnant: Cognizance 8: done"
	on offer
		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed Esquiline station.`
		conversation
			`You walk into Plume's lab and are greeted by a large stack of crates and equipment. Plume and another Remnant are gathered around the stack, checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer. "This is Gavriil, one of our best technologists and a skilled astrobiologist in their own right. They will be going with you."`

> 			`	"It will be a pleasure to work with you," signs Gavriil, making a short bow. "We will be traveling to a research station on <planet>. Yes, it is a gas giant." He smiles at that. "I have to get some equipment set up and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`

Author: ```suggestion
			`	"It will be a pleasure to work with you," signs Gavriil, making a short bow. "We will be traveling to a research station on <planet>. Yes, it is a gas giant." They smile at that. "I have to get some equipment set up and get the core systems online. There will be a few more loads of materials afterwards." They look askance at you. "Shall we get started?"`
```

	name "Transport engineer to <planet>"
	description "An engineer working for Plume needs transportation for himself and some equipment to <planet>."
	blocked "Plume's engineer needs a free bunk and <tons> of cargo space for his equipment. Come back when you have enough space."
	source "Aventine"
	destination "Esquiline"

> 	cargo equipment 8

Author: ```suggestion
	cargo "equipment" 8
```

	on offer
		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed Esquiline station.`
		conversation

> 			`You walk into Plume's lab and are greeted by a large stack of crates and equipment. Plume and another Remnant are gathered around the stack, checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer. "This is Gavriil, one of our best technologists and a skilled astrobiologist in their own right. They will be going with you."`

Author: > Gabriel is male, but androgenous. Mixing terms was an intentional choice to convey that confusion.

Not sure this is a good way to convey that. It just looks like typos.

			`	As you approach <planet>, it occurs to you that the Remnant might appreciate you taking steps to maintain the secrecy of the station. Altering the <ship>'s path slightly, you swing down into <planet> in an oblique path then meander around the planet under the cloud layer until arriving at the station. Your passenger nods approvingly as she notices the flight path. "Secrecy is always good with projects like this. We appreciate the extra effort, and if anyone were to follow you, a path like that through the clouds would leave few signs to spot."`
			`	The <ship> settles into the landing bay, and the technician quickly disembarks. Apparently, there must have been other shipments while you were gone, as there are now a handful of people visible and quite a bit more equipment stacked everywhere. The station itself feels more alive, the air fresher. As before, you promptly receive notification of another payment of <payment> to your account.`

mission "Remnant: Cognizance 11"
	landing
	name "Remnant Data Delivery"
	description "Deliver a data crystal to the engineers on <planet>."
	source "Esquiline"
	destination "Viminal"
	to offer
		or
			has "Remnant: Cognizance 10: done"

> 			has "Remnant: Cognizance 10B: done"

Reviewer: I'm not entirely sure what is going on here, I'll have to re-read this later. (and this is as far as I can review at the moment).

That being said, should that "or" statement be checking for 10A or 10B instead of 10 or 10B? (since you replaced 10 with 10A above)?

Author: > That being said, should that "or" statement be checking for 10A or 10B instead of 10 or 10B? (since you replaced 10 with 10A above)?

I have split 10 into 10A and 10B.
10A is the trip from Esquiline to Caelian, 10B is the trip back to Esquiline, rather than Caelian being a stopover.
11 should require a person to have either done 10 (if they picked up the mission before this change) or 10B.

			`As your ship's sensors re-establish their depiction of the system around you, an alien space station comes into view. It appears to be old and mostly inactive, but your sensors do pick up occasional bursts of energy - and, oddly, faint signs of life almost everywhere.`
	on stopover
		conversation

> 			`Surprisingly, the station is still operational, despite its apparent age. Ancient environmental systems maintain a breathable atmosphere, although it is thinner than you find comfortable. Likewise, the air is cool - but not dangerously so. Everywhere you look, you continually feel like you have stumbled upon a space station built by giants: doors are massive, controls are placed at head height, and displays seem to be angled for a being significantly taller than you. Most unusually, larger rooms often have what look to be benches and tables placed halfway up walls.`

Reviewer 2: Would require adding articles to all of them, and that's a very different tone imo, as it implies a singular set instead of ubiquitous features.

> 			`Surprisingly, the station is still operational, despite its apparent age. Ancient environmental systems maintain a breathable atmosphere, although it is thinner than you find comfortable. Likewise, the air is cool - but not dangerously so. Everywhere you look, you continually feel like you have stumbled upon a space station built by giants: doors are massive, controls are placed at head height, and displays seem to be angled for a being significantly taller than you. Most unusually, larger rooms often have what look to be benches and tables placed halfway up walls.`

Reviewer 3: Good point.

	landing
	name "Deliver navigation data"
	description "Deliver navigational data on the new systems to <destination>."

> 	source "Aventine"

Reviewer: ```suggestion
	source
	 	 government "Remnant"
	 	 neighbor planet "Aventine" "Caelian"
```

This mission was intentionally setup (and written) with the intention that it could trigger on any Remnant location, including stations, and any other permanent bases that are revealed in future content. A better source filter, for now, would be a Remnant planet that is neighbor to Aventine or Caelian. (with both listed, that should enable everywhere in the core trio of systems.

I'm not 100% that I did the neighbor correctly, but hopefully I did. The wiki says multiple planets can be listed in sequence.

Author: If that's what you want, it should probably just be:
```
government "Remnant"
near <any of those three systems> 1
```
But, given that this mission can offer as soon as Cognizance 18 is complete, which occurs on Aventine, there's not much reason to have this mission offer anywhere else.

Reviewer: OK, updated the suggestion above. Thanks!

Author: There's still the fact that this mission will *never* offer anywhere but Aventine anyway, unless you specifically exclude Aventine.

Reviewer: So the fact that it could, theoretically, if the player managed to get there somehow, is in this case more of a safety feature in case someone manages to break something or some corner case we aren't aware of results in it not triggering there this one time.

Unless this is actually a problem, I don't see a reason why allowing it to trigger elsewhere (as a safety feature if nothing else) should be an issue.

Author: What it does is unnecessarily obfuscate what is actually going on. We shouldn't make stuff needlessly complicated as a safety feature in case something breaks, especially when you cannot even describe the mechanism that would have to break. Even if the breakage you suggest occurred, a person would still be able to get this mission upon returning to Aventine anyway.

Reviewer: All it takes is one mission that for some reason takes priority here to block this from happening. 

As far as that goes, I've had this not trigger the first time. No idea why (hasn't happened recently, so I can't provide circumstances or saves), but in my experience the logic for missions firing (especially since the game doesn't let more than one mission offer in the same place at a time) isn't iron-clad.

That being said "any Remnant planet within 2 jumps of Arculus" is hardly obfuscated or hard to understand.

Author: > All it takes is one mission that for some reason takes priority here to block this from happening.

`priority` does not affect landing missions.

The letter thing is well-defined behaviour that people are taking advantage of. Not a vague "if something breaks."

But, actually, I have thought of a reasonable way this could be prevented from offering, though I don't think this arrangement is possible in the vanilla game right now, so sure. I do have other issues with this mission and the following ones which I've outlined in a comment near the top of this, but I don't want to deal with changing that stuff right now.

Reviewer: alright.

		conversation
			branch duskknown
				has "remnant met dusk"

> 			`In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principal difference from before is the presence of five pressure-suited Remnant with a neat stack of equipment topped with helmets. One of them notices you and waves you over. "Captain <first>! My name is Dusk, and I am one of the researchers that you volunteered to transport to Ssil Vida."`

Reviewer 2: ```suggestion
			`In the cafeteria the scene hasn't changed much: there is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principal difference from before is the presence of five pressure-suited Remnant with a neat stack of equipment topped with helmets. One of them notices you and waves you over. "Captain <first>! My name is Dusk, and I am one of the researchers that you volunteered to transport to Ssil Vida."`
```

Looks like you missed one?

mission "Remnant: Cognizance 24"
	name "Equipment to Ssil Vida"
	description "Dusk needs a load of <cargo> picked up from Esquiline, then taken to Ssil Vida."
	source
		government "Remnant"
	stopover "Esquiline"
	destination "Ssil Vida"

> 	cargo "Outpost Equipment" 11

Author: ```suggestion
	cargo "Outpost Equipment" 11
```
```suggestion
	cargo "outpost equipment" 11
```

		event "remnant: ssil vida outfitter"
		conversation
			`Gavriil is waiting at the dock when the <ship>'s hatches finish cycling. "Welcome," they gesture. "It is good to have fresh minds to share <planet> with." The Remnant onboard stream out of the ship in a rush, each carrying a heavy pack of equipment. They gesture respect, and head off in different directions. Evidently they have studied the maps prior to arrival, as they all seem to know where they are going. Several stay behind with Gavriil and help unload the rest of the equipment, some of which appears to include the core equipment of an outfitters.`

> 			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as a base of operations."`

Reviewer 2: ```suggestion
			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as our base of operations."`
```

Iirc, this was actually still proper as it was, but definitely awkward regardless. This would give a bit more ownership of it though.

mission "Remnant: Cognizance 32"
	name "Pick Up Refined Fuel"
	description "Chilia has asked you to pick up 58 tons of refined fuel from <planet> by <date>."
	source "Viminal"
	destination "Clink"

> 	deadline 19

Author: ```suggestion
	blocked "Chilia and Taely look excited about something, but when you inquire, they shake their heads and say you need to have earned their capital license before they can tell you more."
	deadline 19
```

			branch ssilvidaemeraldsword
				has "Sheragi Archaeology: Epilogue: done"
			`	A hologram of Ssil Vida appears in the air over the table, then fades away to leave a large highlighted object with a network of lines reaching around the station. "As Gavriil has told you, the station's primary power source appears to be a massive fusion reactor. As best we can tell, it is quite a simple design, albeit one built for totally automated functioning and bigger than the largest reactors ever built by humanity. Unfortunately, it has consumed its entire fuel supply and is currently using up the last of the fuel it is capable of extracting by refining spent fuel. Our current estimates is that it is producing less than ten percent of its nominal power. This is sufficiently low that it has already ceased powering the majority of the station's systems." He gestures to Chilia.`

> 			`	"Based on our analysis, the reactor appears to use a very specific isotope. Whether it is truly needed or others would suffice is unknown, but for now we are inclined to provide it with an exact match. While we will be able to produce it eventually, it will take time that we do not have." The diagram of Ssil Vida vanishes and is replaced by a map of the galaxy, which focuses in on <system>, near the juncture of the Rim and the South. "Our records indicate that the most easily accessible deposits are on <planet>." Chilia turns to you. "As you are from ancestral space and are most familiar with the people there, we would like you to travel there and bring back 58 tons of processed fuel for the reactor. Can you do this?"`

Reviewer: ```suggestion
			`	"Based on our analysis, the reactor appears to use a very specific isotope. Whether it is truly needed or others would suffice is unknown, but for now we are inclined to provide it with an exact match. While we will be able to produce it eventually, it will take time that we do not have." The diagram of Ssil Vida vanishes and is replaced by a map of the galaxy, which focuses in on <system>, near the juncture of the Rim and the South. "Our records indicate that the most easily accessible deposits are on <planet>." Chilia turns to you. "As you are from ancestral space and are most familiar with the people there, we would like you to travel there and bring back 58 cubic meters of processed fuel for the reactor. Can you do this?"`
```

While the rest of humanity may use tons, the Remnant do not. Given that the word "ton" does, in fact, refer to 1000kg (or 1 cubic meter of water), the reference here to cubic meters does not conflict with the use of ton elesewhere. (Yes, I am well aware that ton *can* refer to about a dozen other potential quantities, but humanity is standardizing on metric, so I strongly suspect that our "ton" is specifically the 1000kg one).

Author: > Given that the word "ton" does, in fact, refer to 1000kg (or 1 cubic meter of water), the reference here to cubic meters does not conflict with the use of ton elesewhere.

The player should not be expected to guess at what the meaning of these units is. If you want to use something other than `tons`, it should just be `units`, since that is a generic term.

Reviewer: There is also the fact that this is within dialog. NPCs are allowed to say things that may confuse the player and/or not conform to the rest of the game.

			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can use without needing hands or arms, it is very limited.`
			`	"Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation.`
			`	"To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`

> 			`	He gestures confidently. "A team of Remnant, for instance, could all review incoming data from an experiment, form their own analyses, and then hold what amounts to a conference where each fully presents their thoughts and ideas with supporting data and reasoning within the span of a few minutes."`

Reviewer 2: This one is a bit tricky.

They are definitely not forming a single common analysis at that point; they each form their own.

But the way it is structured is a bit ambiguous on if it is mentioning the analysis of each individual or the collection of individual analyses.

If the intent is to emphasize them each doing so independently before consulting others, than it is still best to leave it referring to each one's individual analysis.

If the intent is to emphasize the group having the collection to compare, then the change is correct.

> 			`	He gestures confidently. "A team of Remnant, for instance, could all review incoming data from an experiment, form their own analyses, and then hold what amounts to a conference where each fully presents their thoughts and ideas with supporting data and reasoning within the span of a few minutes."`

Reviewer: The point here is each individual Remnant is forming their own individual idea based on the information. So this is referring to plural analyses.
